samples/Hack/EmptyPipelineException.hack

```diff
@@ -1,7 +1,7 @@
-namespace Waffle\Http\Server\Exception;
+namespace Nuxed\Http\Server\Exception;
```

Comment:
Why the changes for the two Hack sample files?

Reply:
the waffle project is deprecated and nuxed uses more hack features, might help in the future when updating the syntax highlight for hack :stuck_out_tongue: ( PHP Syntax highlight is not working well - but its not a priority now )

Reply:
I'd rather we did not change the existing sample files. They are not meant to be execute, but simply to represent in-the-wild usage of Hack.